hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssBaseService.java

```diff
@@ -29,16 +32,23 @@
 import com.swirlds.state.lifecycle.Service;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.time.Instant;
+import java.util.List;
 import java.util.function.BiConsumer;
 import java.util.function.Consumer;
 
 /**
- * Provides the network's threshold signature scheme (TSS) capability and, as a side effect, the ledger id, as this
- * is the exactly the same as the TSS public key.
- * <p>
- * The
+ * Provides the network's threshold signature scheme (TSS) capability and,
+ * as a side effect, the ledger id; as this is exactly the same as the TSS
+ * public key.
  */
 public interface TssBaseService extends Service {
+    /**
+     * Since the roster service has to decide to adopt the candidate roster
+     * based on the available key material, the TSS service must be migrated
+     * before the roster service.
+     */
+    int MIGRATION_ORDER = RosterService.MIGRATION_ORDER - 1;
```

Comment:
nit: We could probably fix one migration order for all services?  Need not be in this PR, just a thought.
So, it will be clear which service happens in which order

Reply:
Interesting idea, a global ordering might well be clearer than these local "relative orderings". 💯